### ISSUE TITLE
Perifissural nodules

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6638]
**Preferred term label:** Perifissural nodules

**Synonyms**
PFN

**Definition (free text, please give PubMed ID)**
Solid, homogenous nodules characterised by a smooth margin and oval, rounded, lentiform or triangular shape. They are typically located within 15 mm from the issue or the pleura. Perifissural nodules may be further differentiated into typical (have contact with interlobar septum) and atypical (do not have contact with interlobar septum). They likely represent intrapulmonary lymphnodes.

**Parent term (use hpo.jax.org/app)**
Pulmonary interstitial high-resolution computed tomography abnormality HP:0025389

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik is this a purely CT finding or can it also be seen on CXR?

**Comment by eswietlik** [comment]
Hi Peter, this is CT finding.

Emilia

śr., 27 sty 2021 o 11:48 Peter Robinson <notifications@github.com>
napisał(a):

**Comment by pnrobinson** [comment]
@eswietlik  This should be a child of Pulmonary nodule? I am adding it there, but please reopen if wrong.
